packages/components/organisms/f-offers/vue.config.js

```diff
@@ -5,6 +5,7 @@ const sassOptions = require('../../../../config/sassOptions')(rootDir);
 
 // vue.config.js
 module.exports = {
+    configureWebpack: { externals: ['@braze/web-sdk', 'vuex', 'js-cookie'] },
```

Comment:
I think we should split this by line for the convenience of adding more externals:
```suggestion
    configureWebpack: { externals: [
        '@braze/web-sdk',
        'vuex',
        'js-cookie'
    ] },
```